twig/extensions/1.0/config/packages/twig_extensions.yaml

```diff
@@ -0,0 +1,10 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+
+    #Twig\Extensions\ArrayExtension: ~
+    #Twig\Extensions\DateExtension: ~
+    #Twig\Extensions\I18nExtension: ~
```

Comment:
this one should not be included IMO, as it conflicts with the Symfony TranslatorExtension

Reply:
removed